ckanext/ozwillo_theme/templates/spatial/snippets/spatial_query.html

```diff
@@ -0,0 +1,29 @@
+{#
```

Comment:
This file is present in ckanext-spatial and I don't see any difference. Why do we need to copy across?  https://github.com/ckan/ckanext-spatial/blob/master/ckanext/spatial/templates/spatial/snippets/spatial_query.html